install/amazon_rds/_01_public.mdx

```diff
@@ -27,13 +24,11 @@ In your [AWS Console](https://console.aws.amazon.com/rds/), modify your existing
 
 Then set the following configuration parameters:
 
-<ImgRdsParamGroup2 />
```

Comment:
[memo] removing https://pganalyze.com/docs/install/amazon_rds/01_configure_rds_instance#enabling-pg_stat_statements image (git animation of changing the following settings), which would be stale with this change